### ISSUE TITLE
ovs setup failed: NotFoundException: Not_found

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#916]
When trying to add a new `master` node with packages `openvstorage-backend`:

```
2016-09-15 16:39:47 06900 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 47 - DEBUG -   172.20.20.131   - Restarting service workers
2016-09-15 16:39:50 59300 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 48 - DEBUG -   172.20.20.131   - Service workers restarted
2016-09-15 16:39:51 71000 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 49 - DEBUG -   172.20.20.201   - Restarting service workers
2016-09-15 16:39:56 01800 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 50 - DEBUG -   172.20.20.201   - Service workers restarted
2016-09-15 16:39:57 17700 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 51 - DEBUG -   172.20.20.231   - Restarting service workers
2016-09-15 16:40:01 59200 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 52 - DEBUG -   172.20.20.231   - Service workers restarted
2016-09-15 16:40:01 66300 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 53 - INFO - Running "extranode" hooks
2016-09-15 16:40:02 84300 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 54 - ERROR - Failed to setup extra node
Traceback (most recent call last):
  File "ovs/lib/setup.py", line 390, in setup_node
    ip_client_map=ip_client_map)
  File "ovs/lib/setup.py", line 1165, in _setup_extra_node
    SetupController._run_hooks('extranode', cluster_ip, master_ip)
  File "ovs/lib/setup.py", line 1777, in _run_hooks
    function(cluster_ip=cluster_ip, master_ip=master_ip, **kwargs)
  File "ovs/lib/helpers/..//albanodecontroller.py", line 351, in model_local_albanode
    main_config = Configuration.get('/ovs/alba/asdnodes/{0}/config/main'.format(node_id))
  File "ovs/extensions/generic/configuration.py", line 112, in get
    data = Configuration._get(key_entries[0], raw)
  File "ovs/extensions/generic/configuration.py", line 362, in _get
    key=key)
  File "ovs/extensions/generic/configuration.py", line 406, in _passthrough
    raise NotFoundException(ex.message)
NotFoundException: Not_found
2016-09-15 16:40:02 84400 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 55 - ERROR - Not_found
Traceback (most recent call last):
  File "ovs/lib/setup.py", line 390, in setup_node
    ip_client_map=ip_client_map)
  File "ovs/lib/setup.py", line 1165, in _setup_extra_node
    SetupController._run_hooks('extranode', cluster_ip, master_ip)
  File "ovs/lib/setup.py", line 1777, in _run_hooks
    function(cluster_ip=cluster_ip, master_ip=master_ip, **kwargs)
  File "ovs/lib/helpers/..//albanodecontroller.py", line 351, in model_local_albanode
    main_config = Configuration.get('/ovs/alba/asdnodes/{0}/config/main'.format(node_id))
  File "ovs/extensions/generic/configuration.py", line 112, in get
    data = Configuration._get(key_entries[0], raw)
  File "ovs/extensions/generic/configuration.py", line 362, in _get
    key=key)
  File "ovs/extensions/generic/configuration.py", line 406, in _passthrough
    raise NotFoundException(ex.message)
NotFoundException: Not_found
2016-09-15 16:40:02 86500 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 56 - INFO - Rolling back setup of current node
2016-09-15 16:40:02 87500 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 57 - INFO - Config management is running
2016-09-15 16:40:02 90000 +0200 - ctrl-grav-02 - 20140/140353625605952 - lib/setup - 58 - DEBUG - Stopping services

```

**Comment by khenderick** [comment]
16.129% chance of hitting this bug. So I'd say "can you please try to reproduce?" ;)

**Comment by khenderick** [comment]
Fixed by #917, packaged in openvstorage-2.7.3-rev.3967.95cd245

**Comment by JeffreyDevloo** [comment]
### Info

We ran into this issue three times in a row which means that our luck ran out (16.129%^3)

### Steps
- Installed OVH twice more
- Didn't encounter this error
- Verified logic with the developer

### Test result

Test passed.
